sklearn/base.py

```diff
@@ -312,15 +312,29 @@ def __setstate__(self, state):
 
     def _get_tags(self):
         collected_tags = {}
-        for base_class in inspect.getmro(self.__class__):
-            if (hasattr(base_class, '_more_tags')
-                    and base_class != self.__class__):
-                more_tags = base_class._more_tags(self)
+
+        if '_more_tags' in self.__class__.__dict__:
+            # we cannot use getattr since it would resolve to the parent's
```

Comment:
we could use ``getattr(type(self), '_more_tags')``, right?

Reply:
no I think it would also resolve to the parents'